### PR TITLE
 fix(cli): Allow `Nd` in man, `nd` in css and `ot` & `stap` in sh

### PR DESCRIPTION
**Pull request by not-my-profile**
Followup PR for #793 (it will be rebased once that PR has been merged so that only the last three commits remain).

### CODE REVIEW

File: crates/typos-cli/src/file_type_specifics.rs [hunk 1]
```diff
@@ -7,6 +7,15 @@
 pub const NO_CHECK_TYPES: &[&str] = &["cert", "lock"];
 
 pub const TYPE_SPECIFIC_DICTS: &[(&str, StaticDictConfig)] = &[
+    (
+        "man",
+        StaticDictConfig {
+            ignore_idents: &[
+                "ND", // .ND macro of groff_ms (see https://linux.die.net/man/7/groff_ms)
```
Comment (epage) [inline crates/typos-cli/src/file_type_specifics.rs:14]: Blocked on #793 

(more note for myself)

**Review by epage — APPROVED** [review]

File: crates/typos-cli/src/file_type_specifics.rs [hunk 2]
```diff
@@ -7,6 +7,15 @@
 pub const NO_CHECK_TYPES: &[&str] = &["cert", "lock"];
 
 pub const TYPE_SPECIFIC_DICTS: &[(&str, StaticDictConfig)] = &[
+    (
+        "css",
+        StaticDictConfig {
+            ignore_idents: &[
+                "nd", // CSS class used by pygments (see https://github.com/pygments/pygments/blob/2.16.1/pygments/token.py#L146)
```
Comment (epage) [inline crates/typos-cli/src/file_type_specifics.rs:14]: This is feeling quite specialized for us to include css classes from specific generators.  People can always put this in their own config

Comment (not-my-profile) [reply crates/typos-cli/src/file_type_specifics.rs:14]: CSS files for Pygment styles are ubiquitous. The GitHub code search [reports 10.1k files](https://github.com/search?q=Name.Decorator+language%3ACSS+.nd&type=code).

Comment (epage) [reply crates/typos-cli/src/file_type_specifics.rs:14]: Eh, if its only css

**Pull request merged**